azure/terraform/modules/app_service/app_service.tf

```diff
@@ -1,78 +1,15 @@
-resource "azurerm_app_service" "app_as" {
+resource "azurerm_linux_web_app" "app_as" {
   name                = local.app_service_name
   resource_group_name = var.app_rg_name
   location            = var.rg_location
-  app_service_plan_id = data.azurerm_app_service_plan.app.id
+  service_plan_id     = data.azurerm_service_plan.app.id
```

Comment:
why has this changed?

Reply:
service_plan_id has been renamed in the new provider version